### DISCUSSION TITLE
Configurable Depth of Field (DOF)

### DISCUSSION THREAD

**Discussion by phreak** [google/model-viewer#2423] [Ideas]
Are there any plans on adding configurable parameters for Depth of Field?

It could contribute to a realistic look:
https://threejs.org/examples/webgl_postprocessing_dof2.html
https://sketchfab.com/3d-models/deep-space-anomaly-71fe27d7e1a64212836646b1f9ca0a98

**Comment by elalish** [comment]
Yeah, we don't have much of a render post-processing story yet. Bloom would be another interesting one. PRs welcome; it's not top of the list since it comes with a significant performance cost.
